static/images/downdeep/dolphin.svg

```diff
@@ -0,0 +1,33 @@
+<svg version="1.1" width="209" height="131" viewBox="1 4 209 131" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
All SVGs should be minimized. I recommend looking at `svgo`:
https://github.com/svg/svgo